package.json

```diff
@@ -37,6 +40,7 @@
     "rollup-plugin-esbuild": "^5.0.0",
     "typedoc": "^0.24.7",
     "typedoc-plugin-missing-exports": "^2.0.0",
+    "typescript": "^5.1.6",
```

Comment:
Had to install typescript since pnpm is a lot stricter than npm (which is a good thing)